readme.md

```
@@ -78,6 +78,201 @@ If a downstream project still uses a cached SNAPSHOT, force updates:
+min=2.345678
+median=2.456789
+p95=2.567890
+max=2.678901
+avg=2.456789
+```
+
+#### Benchmark Results File
+
+Detailed timing data is saved to `cfc-testing/synth_times.txt`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)